xdp-loader/xdp-loader.c

```diff
@@ -139,6 +157,12 @@ int do_load(const void *cfg, __unused const char *pin_root_path)
 			goto out;
 		}
 
+		if (opt->prio)
+			__xdp_program__set_run_prio(p, opt->prio);
+
+		if (opt->actions)
+			__xdp_program__init_chain_call(p, opt->actions);
```

Comment:
This should be a loop, like:

```C
for (i = XDP_ABORTED; i <= XDP_REDIRECT; i++) {
    err = xdp_program__set_chain_call_enabled(p, i, opt->actions & (1U << i));
    if (err)
        goto out;
}
```

This will also correctly override the values loaded from the ELF file...